### ISSUE TITLE
Add a cancel button the UI

### ISSUE THREAD

**Issue by jviotti** [resin-io/etcher#2245]
This will allow the user to cancel all flashes (for now), and potentially particular ones in the future.

**Comment by jviotti** [comment]
This is blocked by https://github.com/resin-io/etcher/issues/2234

**Comment by jviotti** [comment]
We also need some designs from @konmouz

**Comment by lurch** [comment]
Duplicate of #1791 ?

**Comment by jviotti** [comment]
![screen 03-02](https://user-images.githubusercontent.com/2192773/39320474-3c398d04-4952-11e8-91cb-b1fc81eb3dff.png)

**Comment by jviotti** [comment]
We should also add a modal that says "Are you sure you want to cancel?" or something like that

**Comment by konmouz** [comment]
<img width="1210" alt="Screenshot 2019-06-26 at 12 49 22" src="https://user-images.githubusercontent.com/15323961/60177452-d7447f80-9810-11e9-8735-d59d34a6f649.png">

Updated design - https://zpl.io/aRzJezg
